src/Defs.h

```diff
@@ -385,10 +385,10 @@ extern TicksTime g_cycleStartTime;
 extern debug_level_t g_debug_level;
 
 #ifdef  USING_VMA_EXTRA_API
-extern unsigned char* g_pkt_buf;
-extern struct vma_packets_t* g_pkts;
-extern unsigned int g_pkt_index;
-extern unsigned int g_pkt_offset;
+extern __thread unsigned char* g_pkt_buf;
```

Comment:
currently no need to support windows since EXTRA_API is under #if ! defined (WIN32) && ! defined (__FreeBSD__)
https://github.com/Mellanox/sockperf/blob/sockperf_v2/src/Defs.h#L97
i will change them all to on struct